cmd/podman/main_local.go

```diff
@@ -174,13 +174,13 @@ func setupRootless(cmd *cobra.Command, args []string) error {
 	if os.Geteuid() == 0 {
 		ownsCgroup, err := cgroups.UserOwnsCurrentSystemdCgroup()
 		if err != nil {
-			return err
-		}
-		conf, err := runtime.GetConfig()
-		if err != nil {
-			return err
+			logrus.Warnf("Failed to detect the owner for the current cgroup: %v", err)
```

Comment:
Currently users will no see Warnf messages by default.  But this should only happen when running podman in container correct?

Reply:
yes, that is the only case where it should happen